dashboard-service/src/main/java/com/qreal/wmp/dashboard/controller/MainController.java

```diff
@@ -21,14 +23,25 @@
 
     private static final Logger logger = LoggerFactory.getLogger(MainController.class);
 
+    private final UserService userService;
+
     @Autowired
-    private UserService userService;
+    public MainController(UserService userService) {
+        this.userService = userService;
+    }
 
     @RequestMapping("/")
     public ModelAndView home(HttpSession session) {
         logger.info("User {} requested main page.", AuthenticatedUser.getUserName());
 
-        User user = userService.findByUserName(AuthenticatedUser.getUserName());
+        User user = null;
```

Comment:
i'll handle it, but in normal situation user cannot be null, cause to call this method user should be authorized